sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/UserAgentContainerTest.java

```diff
@@ -87,7 +87,7 @@ private String getUserAgentFixedPart() {
         osName = osName.replaceAll("\\s", "");
         String geteUserAgentFixedPart =
             UserAgentContainer.AZSDK_USERAGENT_PREFIX +
-                HttpConstants.Versions.SDK_NAME +
+                "cosmos" +
                 "/" +
                 HttpConstants.Versions.SDK_VERSION +
                 SPACE +
```

Comment:
ditto, for the space, the only constant we should use here is the `HttpConstants.Versions.SDK_VERSION`